tools/system_libs.py

```diff
@@ -1469,14 +1469,11 @@ def warn_on_unexported_main(symbolses):
         return
 
 
-def calculate(temp_files, in_temp, stdout_, stderr_, forced=[]):
-  global stdout, stderr
-  stdout = stdout_
-  stderr = stderr_
+def calculate(temp_files, in_temp, include_libs):
```

Comment:
can just call this `always_include`, the name that is used below? (then it just converts itself into a set)

that name isn't great, but I don't think `include_libs` is an improvement, and the fewer names the simpler to think about at least...